test/spec/provider/camunda-platform/ProcessVariablesProps.spec.js

```diff
@@ -367,24 +368,28 @@ describe('provider/camunda-platform - ProcessVariableProps', function() {
     }));
 
 
-    it('should display items for plane', inject(async function(elementRegistry, selection, canvas) {
+    testForBpmnJsVersion('>=9')('should display items for plane',
```

Comment:
I'm personally a fan of `CHECK && it('...', ...)` to not hide what we're doing.

So to adopt to your example here:

```
bpmnJsVersion(`>=9`) && it('should display items for plane', ...
```

Reply:
Or, if this is a core use-case, simplify it:

```
import {
  bpmnJs9
} from '...'


bpmnJs9() && it('test what works on nine only')
```

Reply:
All just personal preference. Happy to adopt what works for you.

Reply:
Hmm but in that case we hide information that the test case was skipped. WDYT?

Reply:
You got a point on the skip.

Then, to be less verbose why not include a simple `it9` helper?

```
import {
  it9
} from 'test-helper'


it9('do what works with bpmn-js nine+ only');
```

Reply:
Conversely, consider `it10`, `it15`.

Reply:
I shortened this to `withBpmnJs`. `it9` sounds super cryptic to me.